sklearn/linear_model/_base.py

```diff
@@ -419,6 +412,27 @@ class LinearRegression(MultiOutputMixin, RegressorMixin, LinearModel):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
 
+    normalize : bool, default=False
+        This parameter is ignored when ``fit_intercept`` is set to False.
+        If True, the regressors X will be normalized before regression by
+        subtracting the mean and dividing by the l2-norm.
+        If you wish to standardize, please use
+        :class:`sklearn.preprocessing.StandardScaler` before calling ``fit`` on
+        an estimator with ``normalize=False``.
+
+    positive : bool, default=False
+        When set to ``True``, forces the coefficients to be positive. This
+        option is only supported for dense arrays.
+
+        .. versionadded:: 0.24
+           LinearRegression might force the coefficients to be positive.
+
+    maxiter : int, default=None
+        When positive set to ``True`` maximum number of iterations in
+        ``scipy.optimize.nnls``
```

Comment:
```suggestion
        When positive is set to ``True``, indicates the maximum number of iterations in
        ``scipy.optimize.nnls``. Ignored otherwise.
```